src/components/data/EntitiesContent.test.ts

```diff
@@ -0,0 +1,488 @@
+    await user.click(exportButton);
+    const copyMenuItem = screen.getByRole('menuitem', { name: 'Copy to clipboard' });
+    const copyButton = within(copyMenuItem).getByRole('button');
+    await user.click(copyButton);
+
+    // Assert
+    expect(clipboard.writeText).toHaveBeenCalledWith('entity:sample_id\nsample_1\n');
+  });
+
+  it('copies set table to clipboard', async () => {
+    // Arrange
+    const user = userEvent.setup();
+
+    const paginatedEntitiesOfType = jest.fn().mockResolvedValue({
+      results: [
+        {
+          entityType: 'sample_set',
+          name: 'sample_set_1',
+          attributes: {
+            samples: {
+              itemsType: 'EntityReference',
+              items: [
+                {
+                  entityType: 'sample',
+                  entityName: 'sample_1',
+                },
+                {
+                  entityType: 'sample',
+                  entityName: 'sample_2',
+                },
+              ],
+            },
+          },
+        },
+      ],
+      resultMetadata: { filteredCount: 1, unfilteredCount: 1 },
+    });
+    const mockAjax: DeepPartial<AjaxContract> = {
+      Workspaces: {
+        workspace: () => {
+          return {
+            paginatedEntitiesOfType,
+          };
+        },
+      },
+      Metrics: {
+        captureEvent: () => {},
+      },
+    };
+    asMockedFn(Ajax).mockImplementation(() => mockAjax as AjaxContract);
+
+    await act(async () => {
+      render(
+        h(EntitiesContent, {
+          workspace: {
+            ...defaultGoogleWorkspace,
+            workspace: {
+              ...defaultGoogleWorkspace.workspace,
+              attributes: {},
+            },
+            workspaceSubmissionStats: {
+              runningSubmissionsCount: 0,
+            },
+          },
+          entityKey: 'sample_set',
+          activeCrossTableTextFilter: '',
+          entityMetadata: {
+            sample_set: {
+              idName: 'sample_set_id',
+              attributeNames: ['samples'],
+              count: 1,
+            },
+          },
+          setEntityMetadata: () => {},
+          loadMetadata: () => {},
+          snapshotName: null,
+          deleteColumnUpdateMetadata: () => {},
+        })
+      );
+    });
+
+    // Act
+
+    // Select entity
+    const checkbox = screen.getByRole('checkbox', { name: 'sample_set_1' });
+    await user.click(checkbox);
+    screen.getByText('1 row selected');
+
+    // Copy to clipboard
+    const exportButton = screen.getByText('Export');
+    await user.click(exportButton);
+    const copyMenuItem = screen.getByRole('menuitem', { name: 'Copy to clipboard' });
+    const copyButton = within(copyMenuItem).getByRole('button');
+    await user.click(copyButton);
+
+    // Assert
+    expect(clipboard.writeText).toHaveBeenCalledWith('entity:sample_set_id\nsample_set_1\n');
+  });
+
+  it('copies filtered table to clipboard', async () => {
```

Comment:
This is the only test added for this PR; the rest is a carryover from https://github.com/DataBiosphere/terra-ui/pull/4126